app/renderer/#app_status.js#

```diff
@@ -0,0 +1,23 @@
+"use strict";
```

Comment:
Why is there a new file with # in the name?

Please, make sure you remove you editor auto-generated files or add them to .gitignore